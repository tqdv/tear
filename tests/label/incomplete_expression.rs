use tear::twist;

fn bad_expression() {
	// It is a compiler message because we can't detect if the expression is incomplete
	twist! { -label 'a | 1 + }
}

fn main () {}