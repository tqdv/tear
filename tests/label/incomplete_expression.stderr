error: expected expression, found `]`
   --> $DIR/incomplete_expression.rs:5:2
    |
5   |     twist! { -label 'a | 1 + }
    |     ^^^^^^^^^^^^^^^^^^^^^^^^^^ expected expression
    |
   ::: $WORKSPACE/src/twist_impl.rs:167:36
    |
167 |     ( @label-expr ($($flag:tt)*) [ $e:expr ] -> $($l:tt)* ) => {
    |                                    ------- while parsing argument for this `expr` macro fragment
    |
    = note: this error originates in a macro (in Nightly builds, run with -Z macro-backtrace for more info)
